pages/about.html

```diff
@@ -0,0 +1,145 @@
+        >
+          <span class="navbar-toggler-icon "></span>
+        </button>
+        <a class="navbar-brand logo  logoHead" href="../index.html">
+          <img src="../logo.png" alt="no logo found" width="80px" height="50px" >
+          <strong>Web World</strong>
+        
+      </a>
+        <div
+          class="collapse navbar-collapse"
+          id="navbarNav">
+          <ul class="navbar-nav items ">
+            <li class="nav-item liItems">
+              <a class="nav-link active navActive"  aria-current="page" href="../pages/about.html"
+                ><b>About</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Program</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Join</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>Sponsor</b></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#"><b>News</b></a>
+            </li>
+          </ul>
+          <button class="btn btn-outline-success mainBtn" type="button">CC Campaign</button>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container-fluid abCont">
+      <div class="row fstHalf">
+        <img src="../images/bgpic.jpg" alt="no img found" width="100%" height="723px">
+            
+            <img class="abimg" src="../images/abtxt.png" alt="no img found">
+        <span class="abInd position-absolute" ></span>
+        <div class="row abtxt">
+            
+          Web design is made up of several subdisciplines, including user experience (UX) design, user interface (UI) design, and information architecture (IA). 
+            
+        </div>
+        <div class="row">
+          <p class="abtxt2">Please contact us per Email for nay further questions about Web Development!</p>
+          <strong class="abtxt3">faisalakb786@gmail.com</strong>
+        </div>
+      </div>
+
+        <div class="row sndHalf">
+          <h2 class="secHalfHeadind">Web Development Summit 2023 Logo</h2>
+          <span class="abInd"></span>
+          <h5 class="secHalfHeadind2">The logo of Web Development Summit 2023 was decided through the logo competition from 8, June to 7, July.</h5>
+          <img class="abtimg1" src="../images/aboutimg.png" alt="no img found" >
+        </div>
+        <div class="row trdHalf">
+          <h5 class="trdHalfHeading">See the past CC Global Summits</h5>
+          <span class="abInd"></span>
+          <p class="trdPara">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+         <div class="row abTwoimg">
+          <img class="mt-3 abimg1" src="../images/about_2011 1.png" alt="no img found" >
+          <img class="mt-3 abimg1" src="../images/about_2013 1.png" alt="no img found" >
+        </div>
+        </div>
+        <div class="row brandCard1">
```

Comment:
- [ ] As seen in the original design, the sponsors mentioned here should have a dark background color.
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/227392517-fb064428-51a7-4492-8b31-805919125c10.png)
### Original Design
![image](https://user-images.githubusercontent.com/45460192/227392569-c66cbbf5-150a-4c27-b348-4f38b70b3fac.png)

- [ ] Please add a logo to the footer like in the original desing
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/227392733-91d9f852-f0b3-4528-b47d-a89d4756dc60.png)
### Original Design
![image](https://user-images.githubusercontent.com/45460192/227392796-a3811227-cd59-41e4-b3ef-fda9297b2c07.png)